test/Swashbuckle.AspNetCore.SwaggerGen.Test/SwaggerGenerator/SwaggerGeneratorTests.cs

```diff
@@ -48,6 +48,37 @@ public void GetSwagger_GeneratesSwaggerDocument_ForApiDescriptionsWithMatchingGr
             Assert.Equal(new[] { OperationType.Post, OperationType.Get }, document.Paths["/resource"].Operations.Keys);
         }
 
+        [Fact]
+        public void GetSwagger_GeneratesSwaggerDocument_ForApiDescriptionsWithVariousRelativePaths()
```

Comment:
Given the change, I'm not sure this test has much value anymore as it's all just based on direct assignment of the `RelativePath` property surfaced by API Explorer. What do you think?

Reply:
It's not really necessary but I left it in to future-proof. As far as I can tell, it's the only test that validates with optional route params so it gives us coverage for this area in the event of any issues in the future.